lib/common/http-client.ts

```diff
@@ -98,12 +99,15 @@ export class HttpClient implements Server.IHttpClient {
 
 		this.$logger.trace("httpRequest: %s", util.inspect(options));
 
-		const agent = tunnel.httpsOverHttp({
-			proxy: {
-				host: cliProxySettings.hostname,
-				port: parseInt(cliProxySettings.port),
-			},
-		});
+		const agent = cliProxySettings
+			? tunnel.httpsOverHttp({
+					proxy: {
+						host: cliProxySettings.hostname,
+						port: parseInt(cliProxySettings.port),
+					},
+			  })
+			: new Agent({ keepAlive: true }); // If no proxy use default.
+
```

Comment:
```suggestion
		let agent;
		if (cliProxySettings) {
			agent = tunnel.httpsOverHttp({
				proxy: {
					host: cliProxySettings.hostname,
					port: parseInt(cliProxySettings.port),
				},
			});
		}
```

Reply:
I think we don't need to specify a default agent - just pass the `undefined` `let agent;` if no proxySettings. The default Agent doesn't set `keepAlive: true` - so perhaps better to rely on the actual defaults. The Axios docs are a bit misleading about this, but the text above the agent says: 
> This allows options to be added like `keepAlive` that are not enabled by default.

I tested the above suggested change & karma tests run successfully.